internal/cmd/exodus.go

```diff
@@ -48,7 +48,11 @@ func webURI(srcPath string, srcTree string, destTree string) string {
 	// destination paths, see "man rsync" and search for "trailing".
 	if srcTree != "." && !strings.HasSuffix(srcTree, "/") {
 		srcBase := filepath.Base(srcTree)
-		return path.Join(destTree, srcBase, relPath)
+		fileStat, _ := os.Stat(srcTree)
```

Comment:
What's the ignored value?  An error?  We shouldn't be silently ignoring it.

If it's not convenient to deal with an error here, you might consider pushing the "is it a directory" check upwards, e.g. in the function which calls webURI (which seems to be only one place), you check if it's a directory there.  Then pass a srcTreeIsDir argument or something like that into this function.  It should probably be like that anyway, as this is going to stat() the same path for every single file we're dealing with, when we only need to do it once.